src/linuxdeploy.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <iostream>
+#include <boost/filesystem/path.hpp>
+
+#include <linuxdeploy/core/appdir.h>
+#include <linuxdeploy/core/log.h>
+#include <linuxdeploy/util/util.h>
+
+namespace linuxdeploy {
```

Comment:
I don't like utility functions to be in the namespace. Can we locate them outside?

Reply:
It's not meant for utility function but for a major function that encapsulates a part of the workflow. Utility functions should be placed aside.